Consistent protocol version recommendation for 2.10 (#917)
digital-asset/docs.daml.com: docs/2.10.0/docs/canton/usermanual/upgrading.rst

One-Step Migration Recipe for 2.10
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> If you follow this *one-step migration recipe* you're going to:
> 
> - upgrade the Canton binary to a 2.10 release version, and
> - migrate your sync domain to a new sync domain running protocol version 7.

Reviewer: ```suggestion
This recipe allows you to upgrade the Canton binary to release version 2.10 and migrate your sync domain to a new sync domain running protocol version 7.
```

.. note::

>     Version 2.10 supports protocol versions 5 and 7, and protocol version 7 is recommended.
>     Therefore, this recipe states to start a new sync domain running protocol version 7. However,
>     you can use the same instructions to migrate to a new sync domain running protocol version 5,
>     if so desired.

Reviewer: ```suggestion
    Version 2.10 supports protocol versions 5 and 7. Protocol version 7 is recommended and specified in the instructions. However,
    you can use the same instructions to migrate to a new sync domain running protocol version 5,
    if so desired.
```

> The outlined recipe steps have been validated through automated tests to migrate from a sync domain running on a
> :ref:`2.3 <upgrade_to_2.3>`, :ref:`2.4 <upgrade_to_2.4>`, :ref:`2.5 <upgrade_to_2.5>`, :ref:`2.6 <upgrade_to_2.6>`,
> :ref:`2.7 <upgrade_to_2.7>`, :ref:`2.8 <upgrade_to_2.8>` or :ref:`2.9 <upgrade_to_2.9>` release and protocol version
> 3, 4 or 5 to a new sync domain running on the 2.10 release and protocol version 5 or 7.

Reviewer: ```suggestion
The outlined recipe steps have been validated through automated tests to migrate from a sync domain running on release
:ref:`2.3 <upgrade_to_2.3>`, :ref:`2.4 <upgrade_to_2.4>`, :ref:`2.5 <upgrade_to_2.5>`, :ref:`2.6 <upgrade_to_2.6>`,
:ref:`2.7 <upgrade_to_2.7>`, :ref:`2.8 <upgrade_to_2.8>`, or :ref:`2.9 <upgrade_to_2.9>` and protocol version
3, 4, or 5 to a new sync domain running on release 2.10 and protocol version 5 or 7.
```

.. note::

>     If you're using a 2.9 release binary and running a sync domain on protocol version 5, you've got the option

Reviewer 2: I really like the way you phrased this.  Well done.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> General recipe to migrate from a sync domain running on a :ref:`2.3 <upgrade_to_2.3>`,

Reviewer 3: Is it mentioned somewhere that you can use the same instructions to migrate to a 2.10 and pv=5 if desired?

Author: No, that's not mentioned explicitly.

Will revise the text.
